src/test/java/com/kainos/ea/integration/AuthIntegrationTest.java

```diff
@@ -0,0 +1,78 @@
+package com.kainos.ea.integration;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.kainos.ea.WebServiceApplication;
+import com.kainos.ea.WebServiceConfiguration;
+import com.kainos.ea.models.LoginRequest;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class AuthIntegrationTest {
+    private static final DropwizardAppExtension<WebServiceConfiguration> APP =
+            new DropwizardAppExtension<>(WebServiceApplication.class);
+
+    private static final LoginRequest VALID_LOGIN_REQUEST = new LoginRequest(
```

Comment:
we shouldn't store valid creds in code as we'll be quickly hacked. please pass these in an env vars

Reply:
Good point, will create env variables and add to secrets